python/pyspark/sql/types.py

```diff
@@ -498,20 +521,20 @@ def __init__(self, fields=None):
 
     def add(self, field, data_type=None, nullable=True, metadata=None):
         """
-        Construct a :class:`StructType` by adding new elements to it, to define the schema.
+        Construct a StructType by adding new elements to it, to define the schema.
```

Comment:
qq: Why do we change this ?

Reply:
Not needed, seems to be something that have been changed when I was forking the branch 3.1. It should be the same as the current 3.1. I will fix that.